src/styles/main.scss

```diff
@@ -1 +1,47 @@
-// put code here
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  margin: 0;
+  background-color: white;
+}
+
+.watch {
+  position: relative;
+  width: 200px;
```

Comment:
variable?
```suggestion
  width: ...;
```

Reply:
![watch](https://github.com/mate-academy/layout_stop-watch/assets/79792776/e974a28f-2e2a-4eda-acc5-741d6b2230c5)

Reply:
I don't know... when I open it through gh-pages on Github and DEMO LINK, I also have a blank page here...